etc/linux-gnome-shell/README.md

```diff
@@ -0,0 +1,5 @@
+This directory contains a configuration for running syncthing as user startup application in the gnome-shell Linux environment.
+ 1. Copy the file `synthing.desktop` to `$HOME/.config/autostart/syncthing.desktop`
+
+Log output is sent to syslogd.
```

Comment:
Did you test that? I thought there wasn't any logging by default and that you had to add it to the exec comand if you wanted it.

Reply:
@imsodin I haven't tested that, you're right

So should I end up with (as suggested in #755): `Exec=/usr/bin/syncthing -no-browser -no-restart | logger -t syncthing`?

as logging sounds always like a good idea

Reply:
Sorry, I completely forgot about this.  
I don't think we should do any scripting in distributed files - logger may be available on all xdg compatible linuxes (?) but this will e.g. miss logs going to stderr and 2>&1 only works in a shell, so you had to wrap everything into a `sh -c 'syncthing ...` and I am sure there is more pitfalls.  
Unless there is a clear reference, I would either omit any mentioning of logging or change this to a notice that logs may not be stored at all.